File: help/eng_gb/exercicios_da_ficha_4/Ficha04_002/Ficha04_002.js
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("First value \t: " );// print text
    n1 = parseInt(prompt("First value \t: " ,"0") );
    
    document.write(n1 + "\n");// print input 
    document.write("Second value \t\t: " );// print text
    n2 = parseInt(prompt("Second value \t\t: " ,"0") );
    
    document.write(n2 + "\n");// print input 
    var equal = n1 == n2 ;
    document.write("\n" + n1 + "\t==\t" + n2 + "\t=\t" + equal );
    var different = n1 != n2 ;
    document.write("\n" + n1 + "\t!=\t" + n2 + "\t=\t" + different );
    var compare = n1 > n2 ;
    document.write("\n" + n1 + "\t>\t" + n2 + "\t=\t" + compare );
    compare = n1 >= n2 ;
    document.write("\n" + n1 + "\t>=\t" + n2 + "\t=\t" + compare );
    document.write("\n" + n1 + "\t<\t" + n2 + "\t=\t" + ( n1 < n2 ) );
    document.write("\n" + n1 + "\t<=\t" + n2 + "\t=\t" + ( n1 <= n2 ) );
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	domingo, 2022 set 18 23:27

File: help/eng_gb/exercicios_da_ficha_4/Ficha04_002/Ficha04_002.c
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <stdio.h>
#include <string.h>
#include <math.h>
//Main Function
int main() {
    printf("First value \t: ");
    int n1;
    scanf("%d", &n1);
    printf("Second value \t\t: ");
    int n2;
    scanf("%d", &n2);
    int equal = n1 == n2 ;
    printf("\n%d\t==\t%d\t=\t%d",n1 ,n2 ,equal );
    int different = n1 != n2 ;
    printf("\n%d\t!=\t%d\t=\t%d",n1 ,n2 ,different );
    int compare = n1 > n2 ;
    printf("\n%d\t>\t%d\t=\t%d",n1 ,n2 ,compare );
    compare = n1 >= n2 ;
    printf("\n%d\t>=\t%d\t=\t%d",n1 ,n2 ,compare );
    printf("\n%d\t<\t%d\t=\t%d",n1 ,n2 ,( n1 < n2 ) );
    printf("\n%d\t<=\t%d\t=\t%d",n1 ,n2 ,( n1 <= n2 ) );
    return 0;
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	domingo, 2022 set 18 23:27
